index.js

```diff
@@ -35,7 +35,7 @@ const runBsbSync = () => {
 const getBsbErrorMessages = err => err.match(/File [\s\S]*?:\nError: [\s\S]*?\n/g)
 
 const getCompiledFile = (moduleDir, path, callback) => {
-  runBsb((err, res) => {
+  runBsb((err, _, res) => {
```

Comment:
<img width="1130" alt="screen shot 2017-06-14 at 00 04 51" src="https://user-images.githubusercontent.com/32127/27092244-4168639c-5095-11e7-8be6-a24b14b4a992.png">

As screenshot shows, the `BSB check build spec : OK` is not error.